csv-bulk-signature/src/main.ts

```diff
@@ -90,21 +90,53 @@ async function createSignatureFile(
   contact: Contact,
   html: string
 ) {
-  if (fs.existsSync(`${SIGNATURES_PATH}/${fileName}.htm`)) {
-    console.error(
-      `  🔴 ERROR: ${contact["Full Name*"]} already exists. Remove duplicates from CSV and try again.`
-    );
-  } else {
-    await fs.promises.writeFile(`${SIGNATURES_PATH}/${fileName}.htm`, html);
+  const { fullNameFileName, fullName } = getFullNameFileName(contact);
+  const filePath = `${SIGNATURES_PATH}/${fileName}.htm`;
+  const fullNameFilePath = `${SIGNATURES_PATH}/${fullNameFileName}.htm`;
+
+  if (!fs.existsSync(filePath)) {
+    return createFile(filePath, html);
+  }
+
+  if (!fs.existsSync(fullNameFilePath)) {
+    return createFileWithFullName(fullNameFilePath, fullName, fileName, html);
   }
+
+  console.error(
+    `  🔴 ERROR: For ${fullName}, ${fileName}.htm and ${fullNameFileName}.htm already exist.`
+  );
+}
```

Comment:
- refactor to handle one case where fullname is a duplicate too
- show a warning for the first duplicate
- show an error if both options are taken